### PR TITLE
Added better logging, await for the new teams clone to finish  and content cloning!

### PR DESCRIPTION
**Pull request by ScoutmanPt**

### CODE REVIEW

File: scripts/teams-clone-team/README.md [hunk 1]
```diff
@@ -19,71 +22,131 @@ Excelsior, hum? :P
...
 
-    $msg = "`n`r
+    $env:functionName = ""
+    $env:MainFunctionName = ""
+    $msg = "`n`r`n`r
 
-    █▀█ █▄░█ █▀█ █▀▀ ▄▀█ █▄░█ █▀▄ █▄█
-    █▀▀ █░▀█ █▀▀ █▄▄ █▀█ █░▀█ █▄▀ ░█░  `n    MSTeam Builder  `n`n    ...aka ... [team-clone-team]
+       █▀█ █▄░█ █▀█ █▀▀ ▄▀█ █▄░█ █▀▄ █▄█ 
+█▀▀ █░▀█ █▀▀ █▄▄ █▀█ █░▀█ █▄▀ ░█░  `nMSTeam Builder  `n`n    ...aka ... [team-clone-team]
```
Comment (pkbullock) [inline scripts/teams-clone-team/README.md:113]: @ScoutmanPt this isnt rendering think might have some tabs missing, can you validate?

Comment (ScoutmanPt) [reply scripts/teams-clone-team/README.md:113]: Will do!

Comment (ScoutmanPt) [reply scripts/teams-clone-team/README.md]: @pkbullock , whats the best way to address this ? Usually I cancel the pull request and create a new one.

Comment (ScoutmanPt) [reply scripts/teams-clone-team/README.md]: Anyway, i've updated the branch on my side and the pull request is updated. Can you check? @pkbullock

Comment (ScoutmanPt) [reply scripts/teams-clone-team/README.md]: @pkbullock

**Pull request merged**